### PR TITLE
Update steamcmd eggs to use new Yolks images

### PR DESCRIPTION
**Pull request by gOOvER**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

Part 1: I will update all step by step, so its not to much to review

### Changelog:
- updated all eggs to use new yolk source image and debian-installer
- added Auto Update Variable where it's missing
- updated installscripts to latest steamcmd installscript, where is an old one used
- Avorion Egg: removed startup .sh script from startup and use binary. Fix settings.ini download url

### CODE REVIEW

**Comment by gOOvER** [comment]
Install and sturtup of all Servers was successfull

File: game_eggs/steamcmd_servers/ark_survival_evolved/egg-ark--survival-evolved.json [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
     },
-    "exported_at": "2021-07-27T14:22:41+03:00",
+    "exported_at": "2021-11-26T13:48:52+01:00",
     "name": "Ark: Survival Evolved",
     "author": "dev@shepper.fr",
     "description": "As a man or woman stranded, naked, freezing, and starving on the unforgiving shores of a mysterious island called ARK, use your skill and cunning to kill or tame and ride the plethora of leviathan dinosaurs and other primeval creatures roaming the land. Hunt, harvest resources, craft items, grow crops, research technologies, and build shelters to withstand the elements and store valuables, all while teaming up with (or preying upon) hundreds of other players to survive, dominate... and escape! \u2014 Gamepedia: ARK",
     "features": null,
     "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
+        "ghcr.io\/parkervcp\/games:source"
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/ark_survival_evolved/egg-ark--survival-evolved.json:13]: This is a breaking change. There's no RCON cli in this image for the egg to work so it definitely couldn't have passed your startup test

File: game_eggs/steamcmd_servers/7_days_to_die/egg-7-days-to-die.json [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
     },
-    "exported_at": "2019-11-06T20:06:59-05:00",
+    "exported_at": "2021-11-26T13:46:12+01:00",
     "name": "7 Days To Die",
     "author": "kristoffer.norman@bahnhof.se",
     "description": "7 days to die server",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:ubuntu_source",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/games:source"
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/7_days_to_die/egg-7-days-to-die.json:13]: Same as Ark. This image does not have rcon CLI for telnet and as such, breaks this egg.

**Review by Software-Noob — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/ark_survival_evolved/egg-ark--survival-evolved.json (hunk 1 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/ark_survival_evolved/egg-ark--survival-evolved.json:13]: this fix this: https://github.com/parkervcp/yolks/pull/27

**Comment by Software-Noob** [comment]
I'll have to create and test the servers to make sure new changes don't break anything since they run on very old untouched images. I'll try to do this next week when I have more time.

**Comment by Software-Noob** [comment]
As I thought the exec parsing of startup variables doesn't work with script-like startups, causing the ark and 7 days to die eggs to fail. We'll have to update the Yolks images to use eval instead of exec for games:source.

Leaving this here as a note. I'll do it a bit later. The rest of the eggs are fine.

**Review by Software-Noob — APPROVED** [review]
LGTM, thank you for contributing :+1:

**Pull request merged**